scipy/stats/contingency.py

```diff
@@ -1,13 +1,29 @@
-"""Some functions for working with contingency tables (i.e. cross tabulations).
+"""
```

Comment:
Looks like this was intended to be rendered, but I can't find it.

Reply:
I updated the module docstring in `contingency.py` to make Sphinx happy, but in fact we don't generate a separate page for the `contingency` submodule.  I don't think we need one, since everything is already covered in the main `stats` module docstring.